tests/test_pmft.py

```diff
@@ -682,14 +683,76 @@ def test_orientation_with_query_points(self):
 
         # If we swap the order of the points, the angle should no longer
         # matter.
-        pmft.compute((box, points2), angles, points,
-                     neighbors={'mode': 'nearest', 'num_neighbors': 1})
         bond_vector = rowan.rotate(rowan.conjugate(quats), points2 - points)
         bins = ((bond_vector+max_width)*cells_per_unit_length/2).astype(int)
+
+        pmft.compute((box, points2), angles, points,
+                     neighbors={'mode': 'nearest', 'num_neighbors': 1})
         npt.assert_array_equal(
             np.asarray(np.where(pmft.bin_counts)).squeeze(),
             bins.squeeze()[:2])
 
+    def test_orientation_with_fewer_query_points(self):
+        """The orientations should be associated with the query points if they
+        are provided. Ensure that this works when the number of points and
+        query points differ."""
+        boxSize = 8
```

Comment:
🐍 
```suggestion
        box_size = 8
```
or match other parts of the tests that use
```suggestion
        L = 8
```